Web Development/Custom Alert Popup/Readme.md

```diff
@@ -0,0 +1,26 @@
+### Prerequisites
+
+- [X] I am a GSSoC-22 Participant
+- [X] I checked to make sure that this issue has not already been filed
+
+### Description
+
+Hii I'm a GSSoc22 contributor and I'm looking to make a pr request in this repo regarding my project which is a Dice-roller -game.
+
```

Comment:
Provide only the description of the project.

Reply:
Ok allow me I will fix it.